app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -587,6 +585,10 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
     }
 
     override fun showReverseGeocodeFeature(features: List<Feature>) {
+        hideSearchResults()
+        centerOnCurrentFeature(features)
```

Comment:
Alrighty, I'll remove this automatic centering and only center when the device rotates. FYI though- the map doesnt center until after the user lifts their finger and we get a geocoded result to center on.